RetroBar/Languages/русский.xaml

```diff
@@ -66,4 +68,8 @@
     <s:String x:Key="set_time">Настройка _даты/времени</s:String>
 
     <s:String x:Key="open_folder">&amp;Открыть папку</s:String>
-</ResourceDictionary>
\ No newline at end of file
+
+    <FlowDirection x:Key="flow_direction">LeftToRight</FlowDirection>
+
+    <FontFamily x:Key="start_text_xp_font_family">Franklin Gothic</FontFamily>
```

Comment:
These lines are not necessary if they do not affect or if they are already in the [RetroBar/Languages/English.xaml](https://github.com/dremin/RetroBar/blob/master/RetroBar/Languages/English.xaml) file.